Makefile

```diff
@@ -0,0 +1,10 @@
+
+.PHONY: test build
+
+build:
+	go build ./...
+
+test:
+	go install ./cmd/...
```

Comment:
I used the existing steps in the `test.yaml`  and removing this would make the test fail cf [here](https://github.com/heetch/avro/actions/runs/5612860560)

Reply:
:+1: